guidelines/terms/20/audio-description.html

```diff
@@ -13,7 +13,7 @@
       (See also <a>extended audio description</a>.)
    </p>
    
-   <p class="note">Where all of the <a>video</a> information is already provided in existing <a>audio</a>, no additional audio description is necessary.
+   <p class="note">Where all of the important <a>video</a> information is already provided in existing <a>audio</a>, no additional audio description is necessary.
```

Comment:
```suggestion
   <p class="note">Where all of the important visual information is already provided in existing <a>audio</a>, no additional audio description is necessary.
```

Reply:
As discussed on TF call, my inclination is to leave the video>visual change out for now, as tempting as it is. This is just in the interest of tweaking as little as possible. I believe @patrickhlauke will be opening a new issue to consider making this change a bit more globally, and when that's done, we'll mark this suggestion as resolved for now